style/js/shield_defs.js

```diff
@@ -185,10 +197,12 @@ export function loadShields(shieldImages) {
     },
   };
 
+  shields["US:CA:Business"] = banneredShield(shields["US:CA"], ["BUS"]);
```

Comment:
Confirmed these exist.  Unfortunately they were badly tagged, which I fixed, but no ability to make a render without re-rendering.

Reply:
They always seem to be poorly signposted, if at all, so no rush.